### ISSUE TITLE
Register v0.1.0

### ISSUE THREAD

**Issue by OsmanMutlu** [OsmanMutlu/BERT-in-Knet#1]
@JuliaRegistrator register()

**Comment by OsmanMutlu** [comment]
@JuliaRegistrator register()

**Comment by OsmanMutlu** [comment]
@JuliaRegistrator register()

Release notes:

- First version of BERT.jl
- Pretraining module and finetuning module for sequence classification
- BertAdam, which is basically a version of Adam that uses weight decay
- Tokenization is implemented
- Ability to load weights from pytorch models
- Provided examples
